src/easier68k/core/opcodes/lea.py

```diff
@@ -0,0 +1,204 @@
+"""
+>>> str(Lea.from_str('LEA', '(A0), A1')[0])
+'LEA command: src EA Mode: EAMode.ARI, Data: 0, dest EA Mode: EAMode.ARD, Data: 1'
+
+>>> str(Lea.from_str('LEA', '#$0A, A2')[0])
+'LEA command: src EA Mode: EAMode.IMM, Data: 10, dest EA Mode: EAMode.ARD, Data: 2'
+
+>>> Lea.from_str('LEA', 'D3, A3')[1]
+[('Invalid addressing mode', 'ERROR')]
+"""
+from ...core.enum.ea_mode import EAMode
+from ...core.enum.op_size import MoveSize
+from ...core.enum import ea_mode_bin
+from ...simulator.m68k import M68K
+from ...core.opcodes.opcode import Opcode
+from ...core.util import opcode_util
+from ...core.util.conversions import get_number_of_bytes
+from ..util.parsing import parse_assembly_parameter
+
+
+def command_matches(command: str) -> bool:
+    """
+    Checks whether a command string is an instance of this command type
+    :param command: The command string to check (e.g. 'MOVE.B', 'LEA', etc.)
+    :return: Whether the string is an instance of this command type
+    """
+    return opcode_util.command_matches(command, 'LEA')
+
+
+class_name = 'Lea'
```

Comment:
don't think this makes sense to have in the module level

that, and I don't see if/where it is used.

Reply:
It's used in the assembler, because we need to use introspection to get the class itself (which is why it's at the module level): I can probably make it just look for the first class in the module and use that instead.

Reply:
This will be handled in another PR (looks like #79 )